### PR TITLE
#159330953 Completes continuous integration set up

### PR DESCRIPTION
**Pull request by chukwuemekachm**
#### What does this PR do?
- Completes continuous integration set up
#### Description of Task to be completed?
- Install mocha
- Install chai
- Install chai-http
- Install coveralls
- Install nyc
- Install and set up eslint
- Resolve some es6 violations
- Add .travis.yml
- Add .hound.yml
- Add CI badges to README.md
- Add tests for base app in index.test.js
#### How should this be manually tested?
- Clone the repo
- Run `npm install`
- Run `npm run test`
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- #159330953
#### Screenshots (if appropriate)

screen shot 2018-07-27 at 4 51 14 pm

#### Questions:
- None

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
```
Comment (paradisekelechi) [inline .eslintrc.js]: Why use module.exports? @chukwuemekachm

Comment (paradisekelechi) [reply .eslintrc.js]: Switch to eslintrc.json and you would not need to use module.exports

Comment (chukwuemekachm) [reply .eslintrc.js]: eslint don't support es6 imports and exports @paradisekelechi

File: .hound.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+eslint:
```
Comment (paradisekelechi) [inline .hound.yml]: This file is no longer necessary

**Pull request merged**